src/core/Dropdown/Dropdown/Dropdown.baseStyles.tsx

```diff
@@ -1,110 +1,145 @@
 import { css } from 'styled-components';
 import { SuomifiTheme } from '../../theme';
-import { element, inputButton } from '../../theme/reset';
+import { element, input } from '../../theme/reset';
 
 export const baseStyles = (theme: SuomifiTheme) => css`
+  width: 290px;
+  /* stylelint-disable no-descending-specificity */
+  .fi-dropdown_item-list {
+    padding-top: 0;
+  }
   &.fi-dropdown {
     display: inline-block;
 
-    & .fi-dropdown_label--visible {
+    .fi-dropdown_label--visible {
       margin-bottom: ${theme.spacing.xs};
     }
-  }
 
-  & [data-reach-listbox-button].fi-dropdown_button {
-    ${inputButton(theme)}
-    position: relative;
-    display: inline-block;
-    word-break: break-word;
-    overflow-wrap: break-word;
-    min-height: 22px;
-    padding: 7px 38px 7px 7px;
-    border-color: ${theme.colors.depthDark3};
-    text-align: left;
-    line-height: 1.5;
-    background-color: ${theme.colors.whiteBase};
-    box-shadow: ${theme.shadows.actionElementBoxShadow};
-    cursor: pointer;
-
-    &:focus {
-      outline: 0;
-      position: relative;
+    .fi-hint-text {
+      margin-bottom: ${theme.spacing.xs};
+    }
 
-      &:after {
-        ${theme.focuses.absoluteFocus}
+    .fi-status-text {
+      line-height: 18px;
```

Comment:
Should line-height be in rem in case user changes font size in browser settings

Reply:
Fixd